include/aos/common/crypto/mbedtls/cryptoprovider.hpp

```diff
@@ -133,7 +133,8 @@ class MbedTLSCryptoProvider : public aos::crypto::x509::ProviderItf {
 private:
     aos::Error ParseX509Certs(mbedtls_x509_crt* currentCrt, aos::crypto::x509::Certificate& cert);
     aos::Error GetX509CertExtensions(aos::crypto::x509::Certificate& cert, mbedtls_x509_crt* crt);
-    void       GetX509CertData(aos::crypto::x509::Certificate& cert, mbedtls_x509_crt* crt);
+    aos::Error GetX509CertData(aos::crypto::x509::Certificate& cert, mbedtls_x509_crt* crt);
+    aos::Error ConvertTime(const mbedtls_x509_time& src, aos::Time& dst);
```

Comment:
Commit message is too generic. What exactly is fixed?

Reply:
PublicKey export is supposed to be done in a separate PR due to complexity